sdk/appcenter-crashes/src/androidTest/java/com/microsoft/appcenter/crashes/ingestion/models/ErrorModelTest.java

```diff
@@ -301,13 +302,16 @@ public void managedErrorLog() throws JSONException {
                 checkEquals(exception1, exception2);
             }
             {
-                exception1.setStackTrace("1");
+                exception1.setMinidumpFilePath("1");
```

Comment:
do not test 2 fields at once, please keep the existing pattern (and test the new field last)